src/patternfly/components/ChipGroup/examples/chip-group-select-collapsed-example.hbs

```diff
@@ -1,23 +1,23 @@
-{{#> chip-group}}
+{{#> chip-group chip--id="select-collapsed"}}
```

Comment:
should this be `chip-group--id="select-collapsed"` ?

Reply:
good call, changed it to `chip-group--id`